src/pip/_internal/cli/progress_bars.py

```diff
@@ -67,8 +66,7 @@ class InterruptibleMixin:
        download has already completed, for example.
     """
 
-    def __init__(self, *args, **kwargs):
-        # type: (List[Any], Dict[Any, Any]) -> None
+    def __init__(self, *args: List[Any], **kwargs: Dict[Any, Any]) -> None:
```

Comment:
```suggestion
    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

The original annotation was wrong, `*args` and `**kwargs* should annotate the varadic arguments, not `args` and `kwargs` themselves. This was not previous caught because the varadic arguments were only used by `super().__init__()`, which has `type: ignore` due to a Mypy bug.

Reply:
Ok. I will fix that as soon as possible.